hack/versions-management/test/services/test_tags_reconciliation_service.py

```
@@ -0,0 +1,307 @@
+        yield github_client
+
+@pytest.fixture
+def mock_version_repo():
+    with patch("core.services.tag_reconciliation_service.VersionRepository") as mock:
+        version_repo = MagicMock()
+        mock.return_value = version_repo
+        yield version_repo
+
+@pytest.fixture
+def service(versions_file, mock_version_repo, mock_github):
+    with patch("core.services.tag_reconciliation_service.GitHubClient") as mock_gh, \
```

Ruff (0.8.2)

Local variable `mock_gh` is assigned to but never used

Remove assignment to unused variable `mock_gh`

(F841)